### ISSUE TITLE
Allow for entire mods to be considered as "sealed blocks" [REQUEST] 

### ISSUE THREAD

**Issue by Milii20** [micdoodle8/Galacticraft#3999]
Minecraft version: 1.12.2
Galacticraft version: 4.0.2.280

Greetings, today i have, not a bug report nor crash, more a request
if possible, it would be great if we could use some config like:

# List non-opaque blocks from other mods (for example, special types of glass) that the Oxygen Sealer should recognize as solid seals

to allow naming complete mods, like, for example, chisel, to be considered as solid seals, since registering every single block variation from chisel and ztones is really time consuming.

In summary, if saying "chisel:*" for example, would consider every single block from chisel as a "sealed block", it would be great 

Thank you in advance!

**Comment by EzerArch** [comment]
(to devs) As a code example worth seeing, the mod CarryOn (https://github.com/Tschipp/CarryOn) allows such syntax.
